waku/v2/peermanager/peer_connector.go

```diff
@@ -204,7 +204,7 @@ func (c *PeerConnectionStrategy) canDialPeer(pi peer.AddrInfo) bool {
 	val, ok := c.cache.Get(pi.ID)
 	if ok {
 		tv := val.(*connCacheData)
-		now := time.Now()
+		now := time.Now().UTC()
```

Comment:
It would be handy if it's wrapped in a utility function.

Reply:
yup, thought of the same. 
let me see which approach is the best.